rfcs/icon-component.md

```diff
@@ -0,0 +1,69 @@
+# Icon Component
+
+This is the set of ~16 icons used internally within baseui that are safe to open source. This is not the set of uber-specific icons published internally as @uber/icons (though the API is intended to align with that to reduce confusion).
+
+![image](https://user-images.githubusercontent.com/875591/45131331-51e8c500-b141-11e8-898d-ead83d44b345.png)
```

Comment:
do we wan't to have this image in the docs before the usage?

also, i'd use a more descriptive alt text, maybe like `baseui-icons`?